pkg/controller/publishingstrategy/publishingstrategy_controller.go

```diff
@@ -228,15 +225,19 @@ func (r *ReconcilePublishingStrategy) Reconcile(request reconcile.Request) (reco
 					// spec that was generated based on the ApplicationIngress, and the fields checked are immutable,
 					// the actual default IngressController must be deleted
 					reqLogger.Info("Static Spec and Status do not match for default IngressController, deleting")
-					r.client.Delete(context.TODO(), ingressController)
+					if err := r.client.Delete(context.TODO(), ingressController); err != nil {
+						reqLogger.Error(err, "Error deleting IngressController")
+					}
 					return reconcile.Result{Requeue: true}, nil
```

Comment:
Do we not want to return the err here?

Reply:
Possibly, I wanted to preserve the existing behavior (ignoring an error). I can add a TODO here.